packages/syft/src/syft/__init__.py

```diff
@@ -109,6 +109,40 @@
     pass  # nosec
 
 
+try:
+    # third party
+    from IPython.core.guarded_eval import EVALUATION_POLICIES
+
+    ipython = get_ipython()  # type: ignore
+    ipython.Completer.evaluation = "limited"
+    ipython.Completer.use_jedi = False
```

Comment:
I disabled jedi for now, once you use a property getter, it will only be able to autcomplete class level attributes (like methods), not object level attributes (properties). It was way harder to monkeypatch that properly.